torch/nn/init.py

```diff
@@ -274,7 +274,9 @@ def _calculate_fan_in_and_fan_out(tensor):
     num_output_fmaps = tensor.size(0)
     receptive_field_size = 1
     if tensor.dim() > 2:
-        receptive_field_size = tensor[0][0].numel()
+        # math.prod is not always available, accumulate the product manually
```

Comment:
nit: I think it's worth mentioning in this comment that `reduce` is not supported by TorchScript, as that's the true reason for the manual accumulation

Reply:
Good point, will do.